python/tests/test_dataset.py

```diff
@@ -23,13 +37,15 @@ def test_create():
     assert d.coords['y'] == y
     assert d['xy'].data == xy
     assert d['x'].data == x
+    assert d.dims == ['y', 'x']
 
 
 def test_create_from_data_array():
     var = sc.Variable(dims=['x'], values=np.arange(4))
     base = sc.Dataset({'a': var}, coords={'x': var, 'aux': var})
     d = sc.Dataset(base['a'])
     assert d == base
+    assert base['a'] == sc.DataArray(data=var, coords={'x': var, 'aux': var})
```

Comment:
This is a slicing test. Should we rather define `base` as in this line directly as `DataArray`? I believe the original test code is from the time when we did not have `DataArray`, thus the detour via `base['a']`.